README.md

```diff
@@ -1,10 +1,28 @@
 # GitHub Gems: Driving Open-Source Investments With Data
 
-Welcome to the GitHub Gems project! This project hosts a data analytics pipeline that enables smarter investment decisions by measuring the popularity of open-source repos on Github.
+## Project Title and Description
+**GitHub Gems: Driving Open-Source Investments With Data**
 
-## Project Overview
+This ETL (Extract, Transform, Load) project provides an insightful analysis of GitHub open-source projects. Designed for investors and companies considering open-source investments, it offers a detailed view of project dynamics, community engagement, and technological innovation. Data sourced daily from GH Archive is processed and presented in a SQL database format, enabling straightforward querying and advanced analysis.
 
-The goal of this project is to develop an efficient data pipeline that streamlines analytics, reduces manual effort, and enables deeper insights into the open-source ecosystem on GitHub. By leveraging modern data tools and best practices, such as dbt (data build tool) and Airflow, we aim to create a scalable and reliable solution for data-driven decision-making.
+## Key Metrics Targeted
+The ETL pipeline targets an array of metrics to provide a multidimensional understanding of open-source projects:
+
+- **Popularity Metrics:** Tracks the growth rate of repositories based on stars and forks to gauge overall popularity.
+- **Contribution Metrics:** Assesses contributions through commit activities and pull request rates to understand developer involvement.
+- **Influence Score:** A composite metric that integrates various signals such as stars, forks, and issues to evaluate the project's influence.
+- **Developer Engagement Index:** Measures the frequency and recency of contributions, indicating active development and project health.
+- **Issue Resolution Efficiency:** Analyzes how quickly issues are resolved, offering insights into the project's operational efficiency.
+- **Dependency Risk Analysis:** Examines the number and reliability of project dependencies to assess stability risks.
+- **Innovation Rate:** Monitors the introduction of new features and technologies, identifying projects that are at the forefront of innovation.
+- **Funding and Sponsorship Activity:** Tracks sponsorship activities to gauge financial support and commercial interest in the project.
```

Comment:
This list of metrics is great – but kinda overpromises 😂 
How about a "further work" section?

Reply:
Resolved and updated the readme file with the fact and dimension models